app/context.go

```diff
@@ -0,0 +1,100 @@
+package app
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+type ContextKey string
+
+const (
+	TraceIDKey  ContextKey = "trace_id"
```

Comment:
I would say the norm tend to be to use a prefix and not a suffix for block of constants. It helps people with  IDE completion.

Also it helps people to be aware of other constants that exist

Reply:
Done 👍